test/cypress/integration/group-chat-message.spec.js

```diff
@@ -214,7 +214,7 @@ describe('Send/edit/remove messages & add/remove emoticons inside group chat', (
       cy.get('.c-message:nth-child(5) .c-emoticons-list>.c-emoticon-wrapper').should('have.length', 2)
     })
 
-    sendEmoticon(5, '+1', 1)
+    sendEmoticon(5, '+1', 2)
```

Comment:
Why this should be 2? I can not find any changes in test scenario.
Then, why is this changed?

Reply:
As we briefly had a chat about in the slack channel, This is a fix for below test failure. let me keep it for now until things get clarified and hope it sounds good.

<img src='https://user-images.githubusercontent.com/17641213/202371616-84fab717-c1b3-48a6-a4cd-175f22f04a4e.png' width='440'>